chainer/links/activation/maxout.py

```diff
@@ -1,27 +1,46 @@
 import numpy
 
+from chainer import cuda
 from chainer.functions.activation import maxout
 from chainer import link
+from chainer.links.connection import linear
 
 
-class Maxout(link.Link):
+class Maxout(link.Chain):
     """Maxout Networks
```

Comment:
Maxout Networks -> Fully-connected maxout layer. (do not forget the last period)